christfellowshipchurch/web-app-v2
# Hook up connect form to call connect workflow

Workflow is hooked up. When you send a message to a group leader, the success modal will show if it worked correctly.

TO TEST:
See instructions here https://github.com/christfellowshipchurch/apollos-api/pull/206

Release TODO: change hard-coded groupId to be the groupId from props.
